parse/ValueRefPythonParser.cpp

```diff
@@ -350,10 +349,17 @@ void RegisterGlobalsValueRefs(boost::python::dict& globals, const PythonParser&
 
     std::function<boost::python::object(const boost::python::tuple&, const boost::python::dict&)> f_insert_game_rule = [&parser](const boost::python::tuple& args, const boost::python::dict& kw) { return insert_game_rule_(parser, args, kw); };
     globals["GameRule"] = boost::python::raw_function(f_insert_game_rule);
-    std::function<boost::python::object(const boost::python::tuple&, const boost::python::dict&)> f_insert_min = [&parser](const boost::python::tuple& args, const boost::python::dict& kw) { return insert_minmaxoneof_<ValueRef::OpType::MINIMUM>(parser, args, kw); };
-    globals["Min"] = boost::python::raw_function(f_insert_min, 3);
-    std::function<boost::python::object(const boost::python::tuple&, const boost::python::dict&)> f_insert_max = [&parser](const boost::python::tuple& args, const boost::python::dict& kw) { return insert_minmaxoneof_<ValueRef::OpType::MAXIMUM>(parser, args, kw); };
-    globals["Max"] = boost::python::raw_function(f_insert_max, 3);
+
+    // selection_operator
+    for (const auto& op : std::initializer_list<std::pair<const char*, ValueRef::OpType>>{
+            {"OneOf",   ValueRef::OpType::RANDOM_PICK},
+            {"MinOf",   ValueRef::OpType::MINIMUM},
+            {"MaxOf",   ValueRef::OpType::MAXIMUM}})
```

Comment:
not sure if any better / different, but could also do:
```suggestion
    for (const auto& op : std::array<std::pair<const char*, ValueRef::OpType>, 3> {{
            {"OneOf",   ValueRef::OpType::RANDOM_PICK},
            {"MinOf",   ValueRef::OpType::MINIMUM},
            {"MaxOf",   ValueRef::OpType::MAXIMUM}}})
```

Reply:
I use same loop in other places so not sure if forcing to define size is better.